io.openems.edge.goodwe.et/src/io/openems/edge/goodwe/et/ess/Config.java

```diff
@@ -27,6 +27,9 @@
 
 	@AttributeDefinition(name = "Modbus-ID", description = "ID of Modbus brige.")
 	String modbus_id() default "modbus0";
+	
+	@AttributeDefinition(name = "Ess-Capacity", description = "capacity of the battery in Wh")
+	int essCapacity() default 9000;
```

Comment:
-> "capacity"
For readability reasons it's often a good idea to write "9_000".

Reply:
Noted and corrected.